### DISCUSSION TITLE
How do you handle transceiver modules?

### DISCUSSION THREAD

**Discussion by ejm110** [netbox-community/netbox#6507] [Q&A]
1. transceiver modules attached on the device / currently used
2. transceiver modules on stock / not used

**Comment by candlerb** [comment]
I don't track them in Netbox.  We keep a box of transceivers, and order more when it gets low.  If you need to track them:

> 1. transceiver modules attached on the device / currently used

In principle, you'd add them as "Inventory Items" on the device - but that doesn't tell you which interface each one is plugged into.  If you needed that, then I'd suggest you create one or more custom fields on Interface (e.g. for transceiver type), which Netbox v2.11  allows.

> 2. transceiver modules on stock / not used

There's no "spares" functionality in Netbox.  It's not at asset management system at all - it won't manage the lifetime of a particular part which a particular serial number.

Solutions for this range from shared spreadsheet, to installing a separate asset management system, such as [SnipeIT](https://snipeitapp.com/).
